### PR TITLE
Automate OCP-27655

### PR DESCRIPTION
**Pull request by weliang1**
Testing for bug https://bugzilla.redhat.com/show_bug.cgi?id=1796157
Testing Log:
http://pastebin.test.redhat.com/835579

@zhaozhanqi @huiran0826 @rbbratta
Please help to review, thanks!

### CODE REVIEW

File: features/networking/sdn.feature [hunk 1]
```diff
@@ -300,3 +300,52 @@ Feature: SDN related networking scenarios
...
+    And 4 pods become ready with labels:
+      | name=test-pods |
+    And evaluation of `pod(0).name` is stored in the :pod1_name clipboard
+    And evaluation of `pod(0).ip` is stored in the :pod1_ip clipboard
+    And evaluation of `pod(1).ip` is stored in the :pod2_ip clipboard
+    And evaluation of `pod(2).ip` is stored in the :pod3_ip clipboard
+    And evaluation of `pod(3).ip` is stored in the :pod4_ip clipboard
+    And I register clean-up steps:
+    """
+    I run the :delete client command with:
```
Comment (pruan-rht) [inline features/networking/sdn.feature]: @weliang1 you can try the `I ensure...` step instead.

Comment (weliang1) [reply features/networking/sdn.feature]: @pruan-rht Updated it to use:
 Given I ensure "test-rc" replicationcontroller is deleted
 Given I ensure "test-service" service is deleted
Running log: http://pastebin.test.redhat.com/837017

**Pull request merged**